repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/actor.py

```diff
@@ -30,7 +30,8 @@ def process(self):
 
         for rpm_pkgs in self.consume(InstalledRPM):
             for pkg in rpm_pkgs.items:
-                if any(key in pkg.pgpsig for key in RH_SIGS):
+                if any(key in pkg.pgpsig for key in RH_SIGS) or \
+                    (pkg.name == 'gpg-pubkey' and pkg.packager.startswith('Red Hat, Inc.')):
```

Comment:
Indent by additional 4 spaces ([flake8 E128](https://lintlyci.github.io/Flake8Rules/rules/E128.html)).

Reply:
Also add a comment to this line as the reason may not by obvious to everybody.